File: JavaBasic/86_password_match.java
// 86	86_password_match.java	Check if a entered password matches the stored password.
public class _86_password_match {
    public static void main(String[] args) {
        String storedPassword = "secure123";
        String enteredPassword = "secure123";
        System.out.println("Entered: " + enteredPassword);
        if (enteredPassword.equals(storedPassword)) System.out.println("Password Match! Access Granted.");
        else System.out.println("Password Mismatch! Access Denied.");
    }
}

File: JSBASIC/86_password_match.js
// 86	86_password_match.js	Check if a entered password matches the stored password.

let storedPassword = "secure123";
let enteredPassword = "secure123";
console.log("Entered Password:", enteredPassword);
if (enteredPassword === storedPassword) {
    console.log("Password Match! Access Granted.");
} else {
    console.log("Password Mismatch! Access Denied.");
}
